### PR TITLE
Ensure singleLinePerProperty checks each rule on its own

### PR DESCRIPTION
**Pull request by jwilsson**
Props @wareczek (idea taken from an old comment that was deleted for some reason).

Ref #177.

### CODE REVIEW

**Comment by wareczek** [comment]
I figured out that this idea was buggy is some case... but now don't remember it :D

File: lib/linters/single_line_per_property.js [hunk 1]
```diff
@@ -2,8 +2,9 @@
 
 var isValidAfter = function (node, index) {
     var next = node.parent.nodes[index + 1];
+    var types = ['comment', 'rule'];
 
-    if (!next || (next && next.type === 'comment')) {
+    if (!next || (next && types.indexOf(next.type) !== -1)) {
```
Comment (shellscape) [inline lib/linters/single_line_per_property.js]: would Array.includes be faster here?

Comment (wareczek) [inline lib/linters/single_line_per_property.js]: it's ES6 feature :P

Comment (shellscape) [inline lib/linters/single_line_per_property.js]: d'oh!

**Comment by wareczek** [comment]
Fails on:
```css
.modal-content {
    font-weight: normal;
    height: 20px;

    .close {
        background-color: red;
    }
}
```

**Comment by jwilsson** [comment]
Aha, thanks @wareczek!

Do you wanna take another look at it?

**Comment by jwilsson** [comment]
I have a patch ready, just wanted to get your opinion on it, the difference from now is that it'll emit two errors for these kind of lines (one for each property):

```less
.modal-content {
  font-weight: normal; height: 20px;
}
```

I think that's OK. If everyone agrees, I'll update the tests to reflect it (since some of them expect one error now where there will be emitted two instead).

**Comment by wareczek** [comment]
It's ok :)

**Comment by shellscape** [comment]
@jwilsson you've been kicking ass lately!

**Comment by jwilsson** [comment]
@shellscape I wanna get `2.0` out the door ;)

**Pull request merged**